File: app/src/main/java/com/huynd/skyobserver/models/Airport.java
package com.huynd.skyobserver.models;

/**
 * Created by HuyND on 8/11/2017.
 */

public class Airport {
    private String mId;
    private String mName;
    private String mCountryCode;

    public Airport(String id, String name, String countryCode) {
        mId = id;
        mName = name;
        mCountryCode = countryCode;
    }

    public String getId() {
        return mId;
    }

    @Override
    public String toString() {
        return mName;
    }
}

File: app/src/main/java/com/huynd/skyobserver/models/Airport.kt
package com.huynd.skyobserver.models

import android.annotation.SuppressLint
import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Created by HuyND on 8/11/2017.
 */

@SuppressLint("ParcelCreator")
@Parcelize
data class Airport(val mId: String = "",
                   private val mName: String = "",
                   private val mCountryCode: String = "") : Parcelable {

    val id: String
        get() = mId

    fun getCountryCode() = mCountryCode

    override fun toString() = mName
}
